.github/workflows/nightly-build.yaml

```diff
@@ -1,9 +1,9 @@
 {
   name: "Nightly build",
   on: {
-      schedule: [
-          { cron: "05 7 * * *" }
-      ]
+    schedule: [
+      { cron: "05 16 * * *" }
```

Comment:
This'll publish a "nightly" build at 09:05 PT (16:05 UTC) if everything works, but I'll have to merge this PR for the job to run.  Hooray speculative PRs 😞